build/validate.sh

```diff
@@ -145,8 +131,8 @@ if [[ -d "vendor/spryker-eco/$MODULE_NAME/src" ]]; then
     checkCodeSniffRules
     checkPHPStan
 
-    # will be added:
-    #checkDependencyViolationFinder
+    # dev:dependency:find command is not present in spryker-shop/suite
+    # checkDependencyViolationFinder
```

Comment:
we can add conditional code in `checkDependencyViolationFinder` in order to skip it, when command is not available.